### PR TITLE
Add support for 2024 Ford Maverick

### PR DESCRIPTION
**Pull request by GRIFFIT807**
Add fingerprint for:

**Car**
2024 Ford Maverick (non-hybrid) with co-pilot360 assist package

**Route**
5009a9094e925bcc/00000000--c4bc0e701e

### CODE REVIEW

File: selfdrive/car/ford/values.py [hunk 1]
```diff
@@ -139,6 +139,7 @@ class CAR(Platforms):
       FordCarInfo("Ford Maverick Hybrid 2022", "LARIAT Luxury"),
       FordCarInfo("Ford Maverick 2023", "Co-Pilot360 Assist"),
       FordCarInfo("Ford Maverick Hybrid 2023", "Co-Pilot360 Assist"),
+      FordCarInfo("Ford Maverick 2024", "Co-Pilot360 Assist"),
```
Comment (jnewb1) [inline selfdrive/car/ford/values.py:142]:
```suggestion
      FordCarInfo("Ford Maverick 2023-24", "Co-Pilot360 Assist"),
      FordCarInfo("Ford Maverick Hybrid 2023", "Co-Pilot360 Assist"),
```

**Comment by incognitojam** [comment]
Need to update the model year range in https://github.com/GRIFFIT807/openpilot/blob/master/docs/CARS.md too

**Comment by sshane** [comment]
Looks good 

| Dongle, platform, VIN                                                                                                            | Name from VIN 🆚 CarInfo                                                               | Segments              | Bad seg tags   | Good seg tags                                       |
|:---------------------------------------------------------------------------------------------------------------------------------|:---------------------------------------------------------------------------------------|:----------------------|:---------------|:----------------------------------------------------|
| [5009a9094e925bcc](https://useradmin.comma.ai/?onebox=5009a9094e925bcc)<br><sub>FORD MAVERICK 1ST GEN<br>3FTTW8R96........</sub> | FORD Maverick 2024 🆚<br>Ford Maverick 2023<br><sub>🎉 <b>New model year!</b> 🎉</sub> | 3 routes, 15 segments |                | - all lat active: 2<br>- no input all lat active: 1 |

**Comment by sshane** [comment]
I'm assuming this means Co-Pilot360 Assist is correct: https://www.mavericktruckclub.com/forum/threads/latest-2024-maverick-order-guide-released-updated-monthly.34141/

image

@incognitojam think it's worth a test to make sure hybrid and ICE variants match up? I think the cars are similar enough to not fully differentiate them via camera and radar, so with fuzzy FP we might fingerprint on 2024 Maverick Hybrids after this PR for example. Only FW revisions are different for this Maverick hybrid: https://github.com/commaai/openpilot/pull/30467/files

**Pull request merged**